src/Breadcrumb.php

```diff
@@ -357,6 +362,22 @@ public function toArray(): array
         ];
     }
 
+    /**
+     * Constructon helper.
```

Comment:
Please change this description to something like `Helper method to create an instance of this class from an array of data`. Also, as a convention we try to not write comments longer than 80 characters (it's a soft limit anyway, nothing enforced strictly) but if this is the case please consider splitting it into multiple lines to improve readability